plugins/modules/route53_zone.py

```diff
@@ -48,6 +47,16 @@
             - The reusable delegation set ID to be associated with the zone.
             - Note that you can't associate a reusable delegation set with a private hosted zone.
         type: str
+    tags:
+        description:
+            - A hash/dictionary of tags to add to the new instance or to add/remove from an existing one.
+        type: dict
+    purge_tags:
+        description:
+            - Delete any tags not specified in the task that are on the zone.
+              This means you have to specify all the desired tags on each task affecting a zone.
+        default: false
+        type: bool
 extends_documentation_fragment:
 - amazon.aws.aws
 - amazon.aws.ec2
```

Comment:
Could you please add some EXAMPLES that use `tags` and `purge_tags` in the next section, and update the RETURN block for tags?